translations/fa.json

```diff
@@ -423,9 +423,8 @@
     "delete-bootnode-are-you-sure": "آیا مطمئن هستید که میخواهید این bootnode را حذف کنید؟",
     "delete-bootnode-title": "حذف bootnode",
     "delete-category-confirmation": "از حذف این دسته بندی مطمئن هستید؟",
-    "delete-chat": "حذف گفتگو؟",
-    "delete-chat-confirmation": "آیا مطمئنید که می خواهید این چت را حذف کنید ؟",
-    "delete-chat?": "حذف گفتگو؟",
+    "close-chat": "حذف گفتگو؟",
```

Comment:
@jinhojang6 what do you think, is it better to keep key or its better to rename key as well in such cases?